cwf/gateway/fabfile.py

```diff
@@ -58,13 +45,14 @@ def list():
         return list(map(lambda t: t.value, SubTests))
 
 
+@task
 def integ_test(
-    gateway_host=None, test_host=None, trf_host=None,
+    c, gateway_host=None, test_host=None, trf_host=None,
     gateway_vm="cwag", gateway_ansible_file="cwag_dev.yml",
     transfer_images=False, skip_docker_load=False, tar_path="/tmp/cwf-images",
-    destroy_vm=False, no_build=False,
+    destroy_vm=False, build=False,
```

Comment:
was no_build inverted to build? Should the default then be switched to True?

Reply:
In my testing it didn't work to set boolean variables to `False` when calling it from the command line (it just says: `I don't know what 'False' is`). So I changed everything so the default is `False`

It is also bad practice to have negations in the variable names, because if you do logic like `not no_build` it just takes a bit longer to make sure you understand what is going on (technically speaking, `skip_something` is kind of the same in essence, but it feels a bit better, especially for commandline options). I could rename it to `skip_build` if you like that better

Reply:
I just read up again on the invoke manual, and for default `True` variables it automatically creates a `--no-{arg}` flag, i.e. if we have `build=True` as default, we can just add `--no-build` and it sets it to `False`. I will change all the arguments so that the default behavior stays the same.

Reply:
hehe I was just about to ask if they have the --no-VAR syntax :)

I agree that no_build as a variable name is bad practice - but that was not my point. You also changed the if where build is used, i.e., all calls of this function that did use no_build=True do not need the build parameter now and all that did not use no_build now need to use --build. Was this the case?

But the --no-VAR solution is imho better (also in other cases).

Reply:
Yes, I will change it to have the same default settings as before, but have positive variable names (maybe keeping the `skip_unit_tests` because that makes sense semantically). So e.q. that all calls that did use `no_build=True` now use `--no-build` etc.